src/components/controls.vue

```diff
@@ -158,7 +165,7 @@ $controls-transition: 0.3s;
   color: white;
   position: absolute;
   z-index: 10;
-  box-shadow: 5px 5px 8px 0px #0000004a;
+  box-shadow: 5px 5px 8px 0 #0000004a;
```

Comment:
This change is because PHPStorm pointed out the px is unneeded with a 0 measurement.